src/components/shared/TransformNavigator.js

```diff
@@ -24,10 +24,12 @@ type StateProps = $Diff<Props, DispatchProps>;
 const options: ExplicitConnectOptions<{||}, StateProps, DispatchProps> = {
   mapStateToProps: (state: State) => {
     const items = selectedThreadSelectors.getTransformLabels(state);
+    const profileSelection = state.profileView.viewOptions.selection;
     return {
       className: 'calltreeTransformNavigator',
       items,
       selectedItem: items.length - 1,
+      profileSelection: profileSelection,
```

Comment:
As said above, the `uncommitedItem` will be optional so you don't need to change anything here